custom_components/tahoma/const.py

```diff
@@ -9,6 +11,9 @@
 
 DOMAIN = "tahoma"
 
+MIN_UPDATE_INTERVAL = 1
+DEFAULT_UPDATE_INTERVAL = 30
```

Comment:
Do we agree on 30 seconds as default? Or should we keep it 10?

Reply:
I'm ok with this.